packages/prepo-shared-contracts/test/SafeOwnableUpgradeable.test.ts

```diff
@@ -0,0 +1,158 @@
+import { expect } from 'chai'
+import { ethers } from 'hardhat'
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/dist/src/signer-with-address'
+import { ZERO_ADDRESS } from 'prepo-constants'
+import { safeOwnableUpgradeableFixture } from './fixtures/SafeOwnableUpgradeableFixture'
+import { SafeOwnableUpgradeable } from '../types/generated'
+
+describe('SafeOwnableUpgradeable', () => {
+  let deployer: SignerWithAddress
+  let owner: SignerWithAddress
+  let user1: SignerWithAddress
+  let user2: SignerWithAddress
+  let nominee: SignerWithAddress
+  let safeOwnable: SafeOwnableUpgradeable
+
+  const setupSafeOwnableUpgradeable = async (): Promise<void> => {
+    ;[deployer, user1, user2] = await ethers.getSigners()
+    owner = deployer
+    safeOwnable = await safeOwnableUpgradeableFixture()
+  }
+
+  describe('# initial state', () => {
```

Comment:
nit: (`#` only for functions)
```suggestion
  describe('initial state', () => {
```